src/redisearch_rs/inverted_index/src/doc_ids_only.rs

```diff
@@ -21,6 +21,7 @@ pub struct DocIdsOnly;
 
 impl Encoder for DocIdsOnly {
     type Delta = u32;
+    const RECOMMENDED_BLOCK_ENTRIES: usize = 1000;
```

Comment:
I keep remembering this but never get to adding it :sweat_smile: . Thanks :heart: